### PR TITLE
list regular meetings & connection information

### PR DESCRIPTION
**Pull request by mr-c**
- [x] Copy down images to this repo
- [ ] Scale image sizes to improve the layout

### CODE REVIEW

**Review by kinow — APPROVED** [review]
Looking good! Content structure & text is perfect IMO.

File: content/community.md [hunk 1]
```diff
@@ -9,10 +9,46 @@ class: page-short
...
+
+   <img src="https://aws1.discourse-cdn.com/free1/uploads/cwl1/original/1X/3b498f7ee6049bbbfd62ba1ab3d8cd6c7a8179bd.png" alt="screenshot of 'Continue in your Browser' button" />
+4. From the Element native app, or browser session choose "Join" <img src="https://aws1.discourse-cdn.com/free1/uploads/cwl1/original/1X/9c369ca92c995092d8f41c2c8b50f54d3455a60b.png" alt="screenshot of 'Join' button" />
+5. And then choose “Create Account” <img src="https://aws1.discourse-cdn.com/free1/uploads/cwl1/original/1X/c5ae4eba0d06df8d334b4c71e0f23c4c18239edf.png" alt="screenshot of 'Create Account' button" > using your own Google, GitHub, or GitLab, account.
+
+   <img src="https://aws1.discourse-cdn.com/free1/uploads/cwl1/original/1X/9fb8f3d0499ba7c20193e83a43f6606da59489a2.png" alt="screenshot of third party login chooser dialog" />
+6. Now that you’re signed in, join the “CWL Conference” room.
+
+   <img src="https://aws1.discourse-cdn.com/free1/uploads/cwl1/original/1X/3e6d61be4124a2a89fd173c27aea4944a2aa2818.png" alt="screenshot of 'CWL Conference' under 'Suggested Rooms' in the CWL Matrix Space." />
+
```
[the remark this answers is not in the document]
Comment (lunacodes) [reply content/community.md:52]: Took care of this in most recent commit.

File: content/community.md [hunk 2]
```diff
@@ -9,10 +9,46 @@ class: page-short
...
+### Agenda and notes
+
+<https://w3id.org/cwl/meeting_minutes>.
+
+### When
+
+[EMEA](https://en.wikipedia.org/wiki/Europe,_the_Middle_East_and_Africa)-Americas: weekly [every Wednesday at 16:00 UTC from March-November and 17:00 UTC from November-March](https://arewemeetingyet.com/New%20York/2021-07-21/12:00/w/Weekly%20CWL%20Video%20Chat#eyJ1cmwiOiJodHRwczovL21lZXQuaml0LnNpL0NvbXBldGVudEJyYW5jaGVzTm9taW5hdGVXZWFrbHkifQ==)
+
+Asia-Pacific: monthly [every first Friday of every month at 01:00 UTC](https://arewemeetingyet.com/UTC/2022-04-01/01:00/q/Monthly%20CWL%20Video%20Chat)
```
[the remark this answers is not in the document]
Comment (lunacodes) [reply content/community.md]: I updated this in the most recent commit, but just noticed it, but just noticed it has the Outdated tag. Should I undo it?

**Comment by lunacodes** [comment]
@mr-c @kinow I switched the images to be local, and reduced the image sizes (plus added a little vertical spacing). Does the sizing look alright, or should they go smaller still?

**Review by lunacodes — APPROVED** [review]
Approving, provided the image sizes are ok.

**Comment by kinow** [comment]
> @mr-c @kinow I switched the images to be local, and reduced the image sizes (plus added a little vertical spacing). Does the sizing look alright, or should they go smaller still?

Image sizes look good to me using the GitHub UI to preview each file, @lunacodes ! Thanks!

File: content/community.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kinow) [reply content/community.md]: Not sure what tag was changed. I noticed that the EMEA text looks more like what's on the [Google Doc](https://docs.google.com/document/d/1XSAIOe8uHS92IK_MRalwRzF6BNv7bs6pakoYL-C-XTw/edit), by using months. Maybe we need the same for APAC here too? 

>[APAC](https://en.wikipedia.org/wiki/Asia-Pacific) CWL meetings are [every first Friday of the month at 00:00 UTC April-September / 01:00 UTC October-March](https://arewemeetingyet.com/UTC/2022-04-01/01:00/q/Monthly%20CWL%20Video%20Chat) (invite may be updated to match the planned day)

[the remark this answers is not in the document]
Comment (lunacodes) [reply content/community.md]: @kinow In the original excerpt it had `Asia-Pacific: monthly [every first Friday of every month at 01:00 UTC](https://arewemeetingyet.com/UTC/2022-04-01/01:00/q/Monthly%20CWL%20Video%20Chat)`

I updated it to `[APAC](https://en.wikipedia.org/wiki/Asia-Pacific): monthly [every first Friday of every month at 01:00 UTC](https://arewemeetingyet.com/UTC/2022-04-01/01:00/q/Monthly%20CWL%20Video%20Chat)`. Which seems to be what you're asking for(?) - but I was confused by the `Outdated` tag on this comment

**Review by kinow — APPROVED** [review]
@lunacodes I added the months to the APAC meeting (I matched when DST starts/ends here in NZ & AU, same months used in the google docs). Thanks!

**Comment by swzCuroverse** [comment]
Please see my note here: https://github.com/common-workflow-language/cwl-website/issues/126 --- we need to maintain the look and feel of the website and have a process for updating.

**Pull request merged**